File: mvn-advent-of-code/src/test/java/advent/twenty_sixteen/Day11RadioisotopeThermoelectricGeneratorsTest.java
package advent.twenty_sixteen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.*;

public class Day11RadioisotopeThermoelectricGeneratorsTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day11RadioisotopeThermoelectricGenerators("/twenty_sixteen/Day11-RadioisotopeThermoelectricGenerators-input.txt");

        assertEquals(31, dp.getPart1Answer().intValue());
        assertEquals(55, dp.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        DailyProblem<Integer, Integer> dp = new Day11RadioisotopeThermoelectricGenerators("/twenty_sixteen/Day11-RadioisotopeThermoelectricGenerators-example#1.txt");

        assertEquals(11, dp.getPart1Answer().intValue());
        // assertEquals(33, dp.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_sixteen/Day11RadioisotopeThermoelectricGeneratorsTest.scala
package advent.twenty_sixteen

import org.scalatest.{FlatSpec, Matchers}

class Day11RadioisotopeThermoelectricGeneratorsTest  extends FlatSpec with Matchers {
  "2016 Day 11 - Input" should "calculate answers" in {
    val dp = new Day11RadioisotopeThermoelectricGenerators("/twenty_sixteen/Day11-RadioisotopeThermoelectricGenerators-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day11RadioisotopeThermoelectricGenerators("/twenty_sixteen/Day11-RadioisotopeThermoelectricGenerators-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
